README.md

```diff
@@ -888,3 +813,444 @@ Optional.Some(3).tap(x  =>  console.log("Hello") ||  x  +  1);
+Error(3).satisfies(x => x === 3);
+Aborted(3).satisfies(x => x === 3);
+```
+
+#### `Result`'s `map(λ)`
+
+Applies `λ` to your `Result`, returning an `Ok` wrapping the result, if the orignal `Result` is an `Ok`. Otherwise, it evaluates to whatever `Error` or `Aborted` it was applied to.
+
+```javascript
+// Evaluates to Ok(5)
+Ok(4).map(x => x + 1);
+```
+
+#### `Result`'s `mapError(λ)`
+
+Whereas `Result`'s `map` will simply return the original `Error` when applied on an `Error`, `errorMap` transforms an the value wrapped inside the `Error`.
+
+```javascript
+// Evaluates to Error(5)
+Error(4).mapError(x => x + 1);
+
+// Evaluates to 4
+Ok(4).mapError(x => x + 1);
+
+// Evaluates to Aborted('Dead')
+Aborted('Dead').mapError(x => x + 1);
+```
+
+#### `Result`'s `tap(λ)`
+
+Like `map`, but returns the value wrapped by your `Result` unmodified.
+You'd mostly use this if you need the value of your `Result` for side-effects, such as modifying an existing variable that lives outside your `Result` chain, or using the `Result` in an API call, but continuing with the original value, not the return value from the API call.
+
+```javascript
+// Evaluates to 4
+Ok(4).tap(x => x + 1);
+
+// Evaluates to Error
+Error().tap(x => x + 1);
+
+// Evaluates to Aborted
+Aborted().tap(x => x + 1);
+```
+
+#### `Result`'s `flatMap(λ)`
+
+If your `Result` is an `Ok`, `flatMap` maps your `Result` to `λ`, then runs the result through `Result.expect`.
+
+```javascript
+// This evaluates to Error(undefined)
+Ok(null).flatMap(x => x)
+
+// This evaluates to Ok(1)
+Ok(null).flatMap(x => x + 1)
+```
+
+#### `Result`'s `expectMap(λ)`
+
+First maps the `Result` to λ, then runs `expect` on it. The main difference with `flatMap` is that the return value of `λ(value)` is not transformed before being passed to `expect`.
+
+```javascript
+// Evaluates to Ok(['Bearer', 'xyz'])
+Result
+.expect('Bearer xyz')
+.expectMap(R.split(' '))
+```
+
+#### `Result`'s `expectProperty(propertyName)`
+
+When your result is an object, runs an `expect()` on the given property and returns
+that property wrapped in a `Result`.
+
+```javascript
+const header = {
+  'authorization': 'Bearer xyz',
+  'accept-encoding': 'UTF-8',
+};
+
+// This evaluates to Ok('Bearer xyz')
+Result.Ok(header).expectProperty('authorization');
+
+// This evaluates to Error
+Result.Ok(header).expectProperty('x-custom-header');
+```
+
+#### `Result`'s `property(propertyName)`
+
+Extracts a given property from a Result.
+
+```javascript
+const header = {
+  'authorization': 'Bearer xyz',
+  'accept-encoding': 'UTF-8',
+};
+
+// This evaluates to Ok('Bearer xyz')
+Result.Ok(header).property('authorization');
+
+// This evaluates to Ok(undefined) <-- not sure we want this?
+Result.Ok(header).property('authorization');
+```
+
+#### `Result`'s `valueEquals(value)`
+
+Strict compares a value to the value wrapped by the `Result` returning a boolean.
+An `Error` never `equals` any `value` and always evaluates to `false`.
```

Comment:
Neither does Aborted, for that regard.

Also, what happens with `Pending` instances?